api/docs/CMake_rundoxygen.cmake

```diff
@@ -148,7 +148,7 @@ execute_process(COMMAND
   ${DOXYGEN_EXECUTABLE}
   RESULT_VARIABLE doxygen_u_result
   ERROR_VARIABLE doxygen_u_error
-  OUTPUT_QUIET
+#  OUTPUT_QUIET
```

Comment:
Do we want to keep this commented out?

Reply:
Nope, this was the mistake I mentioned offline.  Will remove it.